### PR TITLE
chore(sass): added sass configuration successfully

### PR DESCRIPTION
**Pull request by thomasjosephgreco**
Alright so this is where I am at so far. I was lucky enough to not have to poke around with it too much, but now that we have it going with sass there is a bunch that we can do 😄. 

I am not sure if you want to follow any naming conventions but I am open to whatever. I am keen to styling so I plan on trying out a few different things (possible creating a custom calendar layout using flexbox or something of that nature). 

Also, I am definitely down to add support for a stylelint. If this grows into a Sass-heavy project we could get [sassdoc](http://sassdoc.com/) running as well.

### CODE REVIEW

File: demo/styles/month-view.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+/** 
+** @Calendar Month Styles
+**/
+
+
+.calendar-month-view { 
+
+  .header {
+      text-align: center;
```
Comment (mattlewis92) [inline demo/styles/month-view.scss]: The formatting seems to have gone a bit funky here, can you fix it?

File: demo/styles/week-view.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/** 
+** @Calendar Week Styles
+**/
+@import 'variables';
+
+.calendar-week-view { 
+
+ .day-headers {
+      display: flex;
```
Comment (mattlewis92) [inline demo/styles/week-view.scss]: Same here as well

File: src/calendarMonthView.component.ts [hunk 3]
```diff
@@ -28,6 +28,7 @@ import {
 @Component({
   selector: 'mwl-calendar-month-view',
   template: `
+  <div class="calendar-month-view">
     <div class="month-view">
```
Comment (mattlewis92) [inline src/calendarMonthView.component.ts]: This div can now be removed as it's replaced with `calendar-month-view`

File: demo/main.scss [hunk 4]
```diff
@@ -0,0 +1,2 @@
+@import 'styles/month-view';
+@import 'styles/week-view';
```
Comment (mattlewis92) [inline demo/main.scss]: Can you move all scss files into the root directory, call it scss and rename main to `angular2-calendar` like so:
```
    ├── demo
    └── scss
        ├── angular2-calendar.scss
        ├── month-view.scss
        ├── week-view.scss
        ├── variables.scss
``` 

As demo isn't published to npm so people consuming the module wouldn't be able to import the scss files. This way they can import like:
```
@import 'angular2-calendar/scss/angular2-calendar';
```

**Comment by mattlewis92** [comment]
This looks great! I've also added a couple of nit-picky comments but other than that LGTM! 😄 

My SASS / CSS / sass conventions knowledge is extremely basic, so feel free to rename classes to fit a convention, use mixins or any other sass features which you feel could help improve the flexibility.

**Comment by thomasjosephgreco** [comment]
@mattlewis92 not a problem man. I refactored all of the code to fit with your last review. It should be all ready to rock now 👍. From here I can sift through the .scss files and change any code to adhere to best practices.

**Comment by mattlewis92** [comment]
Awesome, thanks for sorting that out! I look forward to seeing what you come up with for the rest of it 😄

**Pull request merged**